app/helpers/tutors_helper.rb

```diff
@@ -99,9 +100,15 @@ def affiliate
     ]
   end
 
+  def language_proficiencies
+    # Returns a hash like { "English => "None", "Spanish" => "None" }
+    Hash[(native_language.map { |e| e[0] }).collect { |v| [v, 'None'] }]
```

Comment:
Could go either way. Either we store the full hash here or we build it when we display the data, filling in extra pieces at that time. I figured this would be easier since the view could easily consume this data to build the haml, but I can see the argument for storing less data.